### PR TITLE
Feature/document sonarr couchpotato

### PR DESCRIPTION
**Pull request by drmikecrowe**
Please preview and LMK if you would like to include in your readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -111,3 +111,114 @@ Want to contribute? Great!
...
+
+## CouchPotato
+
+Configure CouchPotato with a `blackhole` strategy.  Ensure each will download magnet files.
+
+![CouchPotato Config](images/CouchPotato.png)
+
+## Sonarr
+
+**NOTE**: Take note of the mapping of the `./docker/Downloads/completed:/downloads` directory for Sonarr.  For CouchPotato, we are directly mapping to `./docker/Downloads/completed/movies:/downloads`.  Lesson: Sonarr expects completed downloads to be in the `/downloads/tv` subdirectory (i.e. `tv` category), whereas CouchPotato simply expects completed movies to be in the `/downloads` directory.
```
Comment (cremor) [inline README.md]: Why is this difference needed?

File: README.md [hunk 2]
```diff
@@ -111,3 +111,114 @@ Want to contribute? Great!
...
+
+Add a `blackhole` downloader to Sonarr as follows:
```
Comment (cremor) [inline README.md]: Why did you enable "Read Only"?

File: README.md [hunk 3]
```diff
@@ -111,3 +111,114 @@ Want to contribute? Great!
...
+# Sonarr – TV Show Download and Management
+  sonarr:
+    image: "linuxserver/sonarr"
+    hostname: sonarr
+    container_name: "sonarr"
+    volumes:
+      - ./docker/sonarr:/config
+      - ./docker/Downloads/blackhole/tv:/blackhole
+      - ./docker/Downloads/completed:/downloads
+      - ./NAS/TV:/tv
```
Comment (cremor) [inline README.md]: Note: If the "Read Only" flag in the blackhole configuration is disabled, Sonarr will move (instead of copy) episode files to the TV library. But using separate volumes for `/downloads` and `/tv` will cause Sonarr to be very slow when moving files from the download folder to the TV library.

Explanation: On non Windows systems, Sonarr will always do a copy+delete instead of a normal file system move. (Reason according to a Sonarr developer: "This is to deal with some issues with mono in some file system combinations that lead to truncated files." [Source](https://forums.sonarr.tv/t/manual-import-move-files-actually-copied-then-deleted/23740/2)) This copy+delete is still fast if hard links can be created between those folders. But if the folders are on separate volumes (mounts), this is not possible. In that case Sonarr falls back to a normal (slow) file copy.

TLDR: If the "Read Only" flag in the TorrentBlackhole configuration is disabled (or if the UsenetBlackhole is used, which always moves the items) it's better to have the blackhole watch folder and the TV library on the same volume (mount).

**Comment by chest069** [comment]
I use both of these programs Sonarr, Radarr/Couchpotato is there a way to do this without using it on docker. I am on windows 10 64bit unfortunately. Also sorry for highjacking this I was just wanting to know since I use everything but docker to make things simpler me.

**Comment by cremor** [comment]
@chest069 Sure, should be very easy without docker. Just make sure that all the programs use the same folders. You should be able to find guides with Google.

**Comment by neox387** [comment]
I just use nzbtomedia sonarr option that you would use drone factory ? is deprecated feature..
